x-pack/plugins/observability/common/constants.ts

```diff
@@ -8,7 +8,7 @@
 import { i18n } from '@kbn/i18n';
 
 export const SLO_BURN_RATE_RULE_TYPE_ID = 'slo.rules.burnRate';
-export const OBSERVABILITY_THRESHOLD_RULE_TYPE_ID = 'observability.threshold';
+export const OBSERVABILITY_THRESHOLD_RULE_TYPE_ID = 'observability.rules.threshold';
```

Comment:
Followed the same pattern as SLO. However, I would prefer using `observability.rule.threshold` i.e., `rule` without `s`